File: fix-password.py
import bcrypt

# Generate hash for Test123!
password = b'Test123!'
salt = bcrypt.gensalt(rounds=10)
hash = bcrypt.hashpw(password, salt)

print(f"New hash for Test123!: {hash.decode()}")

# Verify it works
if bcrypt.checkpw(password, hash):
    print("Verification test: PASSED")
else:
    print("Verification test: FAILED")

File: fix-password.js
const bcrypt = require('bcryptjs');

// Generate hash for Test123!
const password = 'Test123!';
const saltRounds = 10;

const hash = bcrypt.hashSync(password, saltRounds);
console.log('New hash for Test123!:', hash);

// Verify it works
const isMatch = bcrypt.compareSync(password, hash);
console.log('Verification test:', isMatch);